src/core_modules/capture-core/src/metaData/Program/Program.js

```diff
@@ -9,6 +9,7 @@ export default class Program {
     _name: string;
     _shortName: string;
     _version: string | number;
+    _categoryCombo: any;
```

Comment:
We could probably create a strong type for CategoryCombination that contains an array of Category where each Category contains an array of CategoryOption.